charm/kafka/actions/upgrade-kafka

```diff
@@ -0,0 +1,12 @@
+#!/usr/local/sbin/charm-env python3
+
+import kafkautils
+import subprocess
+
+from charms.reactive import remove_state
+
+
+remove_state('kafka.nrpe_helper.installed')
+remove_state('kafka.started')
+hookenv.status_set('maintenance', 'upgrade pending')
```

Comment:
Isn't importing `hookenv` required here?

Reply:
It is. Fixed.

Reply:
It is, good catch, fixed.